coffeecard/CoffeeCard.Library/Services/AccountService.cs

```diff
@@ -265,13 +274,22 @@ public async Task RequestAnonymization(User user)
             
             await _emailService.SendVerificationEmailForDeleteAccount(user, verificationToken);
         }
-        
-        public async void AnonymizeAccount(string token)
-        {
-            Log.Information($"Trying to verify deletion with token: {token}");
 
-            var user = VerifyTokenClaimAndUser(token);
+        public async Task AnonymizeAccountAsync(string token)
+        {
+            Log.Information("Trying to verify deletion with token: {token}", token);
 
+            User user;
+            try
+            {
+                user = await VerifyTokenClaimAndUserAsync(token);
+            }
+            catch (ApiException e)
+            {
+                // No log, since the ApiException should have been logged on throwing
+                return;
+            }
```

Comment:
Why do we supress the exception? I assume we want to return the 401 to the user?

Reply:
Unless we want to suppress he exception, I suggest to remove the catch block since we are not changing logic if exception is thrown

Reply:
Hmm, the reasoning for it was this method is only called from the webpages, so to limit the amount of exceptions they receive. This was probably the wrong approach, and the better way would be to catch the ApiException separately on the webpages and handle it by showing an appropriate error message

Reply:
I agree with what you suggest as your solution. Ignore it here, catch at web pages 😉